### ISSUE TITLE
MiniProfiler prevents linq2db from saving some datatypes to PostgreSQL

### ISSUE THREAD

**Issue by evilbloodydemon** [linq2db/linq2db#942]
MiniProfiler prevents linq2db from saving some datatypes to PostgreSQL.
As I understand, provider tries to find required types in assembly where connection type is, assuming that it is Npgsql. But instead searching is performed in MiniProfiler assembly.

### Steps to reproduce

Add MiniProfiler to your project according to readme.
Add table with boolean field and model for it
Try to save data

Result: 
```
System.NullReferenceException: Object reference not set to an instance of an object.

Line 254:				case DataType.Boolean    : _setBoolean  (parameter);                   break;
Source File: c:\projects\linq2db\Source\DataProvider\PostgreSQL\PostgreSQLDataProvider.cs    Line: 254 
```

_setBoolean turns out to be null (so do _setJsonb, _setJson, etc)

### Environment details
linq2db version: *1.10.0*  
Database Server: *PostgreSQL 9.6.3*  
Database Provider: *Npgsql 3.2.5*  
Operating system: *Windows 10*

**Comment by evilbloodydemon** [comment]
#360 seems to be related

**Comment by MaceWindu** [comment]
duplicate of #1438

**Comment by MaceWindu** [comment]
closed by #1447
